src/main/sensors/acceleration.h

```diff
@@ -54,10 +59,11 @@ typedef enum {
 typedef struct acc_s {
     accDev_t dev;
     uint16_t sampleRateHz;
-    vector3_t accADC;
+    vector3_t accADC;                       // rotated but unscaled ADC value
+    vector3_t jerk;
+    float accMagnitude;                     // in multiples of 1G
```

Comment:
`accMagnitude` is used at least 3 times currently. Thinking about new features in the foreseeable future increases use cases even more. Same for `jerkMagnitude`. So I think this can potentially save quite some cycles to cache them in every loop.